tests/Umbraco.Tests.Integration/Umbraco.Tests.Integration.csproj

```diff
@@ -6,19 +6,27 @@
     <Description>Contains helper classes for integration tests with Umbraco CMS, including all internal integration tests.</Description>
     <RootNamespace>Umbraco.Cms.Tests.Integration</RootNamespace>
     <IsPackable>true</IsPackable>
-    <EnablePackageValidation>false</EnablePackageValidation>
-    <NoWarn>NU5100</NoWarn>
+    <EnablePackageValidation>$(BaseEnablePackageValidation)</EnablePackageValidation>
```

Comment:
Any projects that is distributed as a package should have package validation enabled (use the setting value defined in `Directory.Build.props`). The `Umbraco.Cms.Tests.Integration` package contains public helper classes, which shouldn't contain breaking changes within the the current major!

However, all integration tests should be excluded by this, so maybe we can make all tests internal and not publicly expose them in the shipped package? Again, this is something to fix in a separate PR...